src/sentry/integrations/issues.py

```diff
@@ -235,6 +235,26 @@ def create_default_repo_choice(self, default_repo):
         """
         return (default_repo, default_repo)
 
+    def get_annotations(self, group):
+        external_issue_ids = GroupLink.objects.filter(
+            group_id=group.id,
+            project_id=group.project_id,
+            linked_type=GroupLink.LinkedType.issue,
+            relationship=GroupLink.Relationship.references,
+        ).values_list('linked_id', flat=True)
+
+        external_issues = ExternalIssue.objects.filter(
+            id__in=external_issue_ids,
+            integration_id=self.model.id,
+        )
```

Comment:
I'm guessing django doesn't give us a reasonable way to query both these tables with one query or a subquery?

Reply:
Hmm.. outside of using `extra` I'm not sure there is a better way to do that. Though I would be careful with extra subqueries and mysql though... different versions don't support it if I recall correctly. I feel this is pretty clear and wouldn't advocate for going that route.